sample_file.py

```
@@ -0,0 +1,47 @@
+USERNAME = "admin2"
+PASSWORD = "password123"
+
+import os
```

Ruff

Module level import not at top of file

(E402)